packages/snaps-controllers/src/snaps/registry/json.test.ts

```diff
@@ -352,4 +352,29 @@ describe('JsonSnapsRegistry', () => {
       expect(result).toBeNull();
     });
   });
+
+  describe('updateDatabase', () => {
```

Comment:
Nit: Since it's calling `SnapsRegistry:update`, maybe this makes more sense.
```suggestion
  describe('update', () => {
```